src/core/update_initial_conditions.jl

```diff
@@ -136,7 +130,7 @@ function status_init(
         psi_container,
         devices,
         ICKey(DeviceStatus, T),
-        _make_initial_condition_active_power,
+        _make_initial_condition_status,
         _get_status_value,
     )
```

Comment:
@jd-lara, I think we need a cache here. This works for a single stage simulation, but with a UC->ED, it fails because it can't find the ON variable in the ED stage when initializing the 2nd UC step.